pkg/fakes/fakes.go

```diff
@@ -0,0 +1,55 @@
+package fakes
```

Comment:
The pattern I have seen around is to nest the fakes under the main implementation, could that work or might introduce cyclic deps?

Reply:
Let me try that 👍

Reply:
Works fine 👌